### PR TITLE
Integrate playbooks with CDI

### PR DESCRIPTION
**Pull request by rthallisey**
- Initial playbooks for cdi copied over from kubevirt-ansible
- Makefile
- Move Dockefile to directory root

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM quay.io/water-hole/ansible-operator
```
Comment (tiraboschi) [inline Dockerfile]: The command
  operator-sdk build quay.io/example/cdi-operator:v0.0.1
does almost this without the need of an embedded Dockerfile.
Do we prefer to maintain our own Docker file or to require the operator-sdk at build time?

**Comment by tiraboschi** [comment]
We can download the role code from kubevirt-ansible repo at build time with:
  wget -qO- https://api.github.com/repos/kubevirt/kubevirt-ansible/tarball/master | tar xzv  --wildcards "*/roles/cdi" --strip-components=1

File: Dockerfile (hunk 1 above)
Comment (rthallisey) [reply Dockerfile]: I don't think we need to.  The base Dockefile uses ansible runner, which enables tower integration and I don't think we need to customize the run command.

**Comment by rthallisey** [comment]
> Still not sure if we prefer to keep the CDI deployment role in ansible-playbook or just copy here.
On my opinion the worst case is definitively having it here and there with the need of keeping them aligned.
>

I'm still open to downloading from kubevirt-ansible, but for a first pass let's copy it in.  I think there are some additional challenges that we will encounter which will change the location and makeup of the playbook.  For now, it's easiest to hold it here.

**Pull request merged**